GITMCode/GITM
# [test]: better tests scripts! and ref solutions

# [test] Add more functionality to test script

In essence:
- Test script can be run on all, or one, UAM file
- Check outputs agaionst reference solutions
- Update reference solutions
- Better help 
- Support incorrect command line flags.
